### PR TITLE
Added Travis CI support to OpenSSL1.1 branch

### PR DESCRIPTION
**Pull request by fmonteiro93**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+services:
+  - docker
+
+before_install:
+  - docker pull fmonteiroctw/mococrw_build_env
+  - docker images
+
+script:
+  - mkdir build
+  - docker run --rm -t -v "$(readlink -f .):/src" -v "$(readlink -f build):/build" --workdir /build fmonteiroctw/mococrw_build_env cmake -DCMAKE_BUILD_TYPE=Coverage -DBUILD_TESTING=On /src
```
Comment (neverpanic) [inline .travis.yml]: Can we make `fmonteiroctw/mococrw_build_env` a container that's automatically built from a certain source, so anybody can rebuild it at a later point in time?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  - docker run --rm -t -v "$(readlink -f .):/src" -v "$(readlink -f build):/build" --workdir /build fmonteiroctw/mococrw_build_env make -j2
+  - docker run --rm -t -v "$(readlink -f .):/src" -v "$(readlink -f build):/build" --workdir /build fmonteiroctw/mococrw_build_env ctest -j2 --output-on-failure
```
Comment (neverpanic) [inline .travis.yml]: Newline at EOF, please.

File: .travis.yml (hunk 1 above)
Comment (fmonteiro93) [reply .travis.yml]: Ageed. Linked github repository to the dockerhub repository so further builds can be made.

File: .travis.yml (hunk 2 above)
Comment (fmonteiro93) [reply .travis.yml]: Done.

File: .travis.yml (hunk 1 above)
Comment (neverpanic) [reply .travis.yml]: Ah, it's only visible with login, so maybe that was there before and I just missed it.

**Pull request merged**